### PR TITLE
Adds add multi-key test

### PR DESCRIPTION
**Pull request by jeremy-then**

### CODE REVIEW

File: rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java [hunk 1]
```diff
@@ -2464,6 +2464,52 @@ void voteFederationChange_addWrongFederatorPublicKey_returnsGenericResponseCode(
 
         }
 
+        @Test
+        void voteFederationChange_addFederatorMultiKey_returnsExpectedPendingFederationSizeResponseCode() {
```
Comment (julia-zack) [inline rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java:2468]: `_returnsSuccessful`. And pending federation size response code does not exist

Comment (jeremy-then) [reply rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java]: Updated.

File: rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java [hunk 2]
```diff
@@ -2464,6 +2464,52 @@ void voteFederationChange_addWrongFederatorPublicKey_returnsGenericResponseCode(
...
         }
 
+        @Test
+        void voteFederationChange_addFederatorMultiKey_returnsSuccessfulResponseCode() {
+
+            // Arrange
+
+            BtcECKey expectedBtcECKey = new BtcECKey();
+            ECKey expectedRskKey = new ECKey();
+            ECKey expectedMstKey = new ECKey();
```
Comment (marcos-iov) [inline rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java:2474]: Use deterministic values please

Comment (marcos-iov) [reply rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java:2474]: And also meaningful names, those are the keys of a new federation member. Calling them "expected keys" doesn't give much info

Comment (jeremy-then) [reply rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java]: Updated.

**Review by marcos-iov — APPROVED** [review]

**Pull request merged**